auth-web/src/components/pay/eft/ShortNameAccountLink.vue

```diff
@@ -216,6 +259,14 @@ export default defineComponent({
       emit('on-link-account', account)
     }
 
+    async function unlinkAccount (item) {
```

Comment:
Added in basic unlinking to make things easier to test. 

Upcoming unlinking tickets will add in error handling / modals